src/common/mfu_daos.c

```diff
@@ -1362,6 +1361,10 @@ daos_args_t* daos_args_new(void)
     da->daos_preserve = false;
     da->daos_preserve_path = NULL;
 
+    /* in most cases source only needs RO and destination needs RW */
+    da->default_src_cont_open_flags = DAOS_COO_RO;
```

Comment:
What was the side effect of not setting permissions on source and destination.
Does that mean the delete never worked in days.

Reply:
Yep! We used to set RW on the source and destination, so remove worked. Then we trimmed just the source to be RO, which broke remove.